usds/justice40-tool
# Adds matplotlib to the pyproject.toml for score/ to avoid import error

## Summary

Fixes #322 by adding `matplotlib` to the project requirements for `score/`

## Changes proposed

- Adds `matplotlib` (and dependencies) to the following files:
  - `pyproject.toml`
  - `poetry.lock`
  - `requirements.txt`

## Steps to review

1. Conduct a fresh install of the `score/` directory following the instructions in #322 to reproduce the error
2. Checkout this PR then re-run `poetry install` to get the new dependencies 
3. Open up a python interpreter: `poetry run python`
4. Run `import matplotlib` in the python interpreter and you should no longer receive the `ModuleNotFoundError`
